lib/docker-delta.coffee

```diff
@@ -10,7 +10,7 @@ btrfs = require './btrfs'
 utils = require './utils'
 Docker = require 'docker-toolbelt'
 
-docker = new Docker()
+docker = new Docker(socketPath: process.env.DOCKER_SOCKET ? '/var/run/docker.sock')
```

Comment:
No, this is a generic docker module. The default will stay the same as docker's default which is `/var/run/docker.sock`